js/common.js

```diff
@@ -0,0 +1,8 @@
+const $ = (select, el = document) => el.querySelector(select);
+
+const SIGN_INFO_REGEX = {
+  email: /^[A-Za-z0-9_\.\-]+@[A-Za-z0-9\-]+\.[A-Za-z0-9\-]+/,
+  password: /^[A-Za-z0-9]{8,15}$/,
```

Comment:
상수의 경우 객체의 key값도 대문자로 표현되면 좋을 것 같네요!

```suggestion
  EMAIL: /^[A-Za-z0-9_\.\-]+@[A-Za-z0-9\-]+\.[A-Za-z0-9\-]+/,
  PASSWORD: /^[A-Za-z0-9]{8,15}$/,
```